ladybug/comfort/standard/pmv.py

```diff
@@ -0,0 +1,406 @@
+# coding=utf-8
+"""Object for calculating PMV comfort from DataCollections."""
+from __future__ import division
+
+from .comfortmodel import ComfortModel
+from .comfortmodel import ComfortParameter
+from ...datacollection import DataCollection
+from ...epw import EPW
+from ...psychrometrics import humid_ratio_from_db_rh
+
+
+class PMV(ComfortModel):
+    """PMV Comfort Object."""
+
+    def __init__(self, air_temperature, rel_humidity,
+                 rad_temperature=None, air_speed=None,
+                 met_rate=None, clo_value=None, external_work=None,
+                 comfort_parameters=None):
+        """Initialize a PMV comfort object from DataCollections of PMV inputs.
+
+        Args:
```

Comment:
Arguments description is missing!

Reply:
Again, this class is not finished.  See comment above.

Reply:
I filled in some Arg docstrings anyway.